.github/workflows/unit-test.yml

```diff
@@ -134,10 +133,12 @@ jobs:
     env:
       NPM_CONFIG_UNSAFE_PERM: true
     steps:
-      - name: Checkout
-        uses: actions/checkout@v1
       - name: Permission Setup
         run: sudo chmod -R 777 /github /__w
+      - name: Checkout
```

Comment:
checkout moved to after the permission setup. We use the circleci container because it has chrome preinstalled, but it has weird permission issues.